### DISCUSSION TITLE
Need help in Proposals

### DISCUSSION THREAD

**Discussion by rghdrizzle** [cncf/mentoring#625] [Q&A]
Hey , im luqmaan , i am kinda new to open source and it is my first time participating in GSOC , i need some guidance on how I should write a proposal and how I should proceed with the project . Any help would be appreciated :)

**Comment by 0904-mansi** [comment]
Hey @rghdrizzle for proposal writing you can watch some videos on youtube and here is the official proposal format by GSoC https://google.github.io/gsocguides/student/writing-a-proposal you can also check some previous year's accepted proposals here https://github.com/prondubuisi/accepted-gsoc-proposals 
Hope it helps:)
